core/src/consensus.rs

```diff
@@ -29,7 +29,7 @@ pub const REWARD: u64 = 1_000_000_000;
 /// Number of blocks before a coinbase matures and can be spent
 /// TODO - reduced this for testing - need to investigate if we can lower this in test env
 // pub const COINBASE_MATURITY: u64 = 1_000;
-pub const COINBASE_MATURITY: u64 = 10;
+pub const COINBASE_MATURITY: u64 = 3;
```

Comment:
We should track this somewhere so we don't forget it.

Reply:
Opened an issue to think about making this configurable based on mining mode - #133 
and to document this somewhere.